Docs/details/scopes.md

```diff
@@ -0,0 +1,27 @@
+# Scopes
+
+In addition to rules, Santa can whitelist or blacklist based on a scope. There are currently only a few scopes implemented. Scopes fall into one of two categories, a whitelist scope or a blacklist scope. Scopes are evaluated after rules. The blacklist scopes are evaluated before the whitelist scopes.
+
+Scopes are simply less specific or broader way of whitelisting or blacklisting `exec()`.
+
+For configuration of scopes see the [configuration.md](../deployment/configuration.md).
```

Comment:
For configuration of scopes see [configuration.md](../deployment/configuration.md).